src/pages/payment-methods/index.tsx

```diff
@@ -226,9 +226,8 @@ const DisplayAccordion = ({ locale }: PaymentMethodsProps) => {
                     return null
                 } else
                     return (
-                        <Accordion has_single_state>
+                        <Accordion key={idx} has_single_state>
```

Comment:
can we use any other key on this?
idx is not a safe key

Reply:
Thanks man, explained in a direct message my opinion, changed the key 👍